contracts/colony/ColonyDataTypes.sol

```diff
@@ -349,6 +349,18 @@ interface ColonyDataTypes {
 
   event ArbitraryTransaction(address target, bytes data, bool success);
 
+  /// @notice Event logged when an expenditure payout is claimed.
+  /// @dev This is emitted in addition to the other PayoutClaimed
+  /// event. The other will be removed soon.
+  /// @param agent The address that is responsible for triggering this event
+  /// @param id Id of the expenditure
+  /// @param slot Expenditure slot of the payout claimed
+  /// @param token Token of the payout claim
+  /// @param tokenPayout Amount of the payout claimed, after network fee was deducted
+  event PayoutClaimed(address agent, uint256 id, uint256 slot, address token, uint256 tokenPayout);
```

Comment:
IIRC we discussed emitting the payout before the network fee is deducted. Is that still the plan?

Reply:
That's the existing implementation, it's just that this comment is wrong from my copy/pasting. But I want to know why we did it the other way in the original implementation, plus whether this is used in the existing stack.

Reply:
Ok, standing by

Reply:
Any update?

Reply:
It is indeed [used](https://github.com/JoinColony/block-ingestor/blob/0a4a9db0f0241be6e8d0855e7954d17fe614dc06/src/handlers/actions/payment.ts#L71-L72) [in](https://github.com/JoinColony/colonyCDapp/blob/d10adabb5986edb0fd6deb481a7fd007ae640d3e/src/i18n/en-events.ts#L48) the existing stack, so I think we need to deduct the fee.

In terms of why we did it this way, I have not been able to dig it up. My best guess is we decided the amount being actually received was the amount most naturally of interest, but I do agree that is already available via the ERC20 events.